cime_config/testdefs/testmods_dirs/clm/mimics/include_user_mods

```diff
@@ -0,0 +1 @@
+../coldStart
```

Comment:
Since this is inheriting the coldStart the name of the testmod should include that in the name. So mimicsColdStart or at least mimicsCold. At some point we should have a mimics option that doesn't do a cold-start and provides a spunup initial condition file to use.

Reply:
I think it makes sense to switch to spun-up initial conditions when we have an acceptable finidat file, so I will leave the mods directory name unchanged if that's ok. I will include a comment to that effect in the README.

Reply:
The comment that I added to this testmod's README:

```
As of 2021/11/15 this test inherits mods from the ../coldStart directory;
however the plan is to change that to ../default when we have an acceptable
spun up finidat file to initialize MIMICS with.
```